lib/build.js

```diff
@@ -11,40 +17,40 @@ module.exports = function ({
   minify,
   watch
 }) {
-  function buildJs ([entry, outfile]) {
-    const pipeline = browserify({
-      config,
-      entry,
-      env,
-      minify,
-      outfile
-    })
+  return files.map(([entry, outfile]) =>
+    entry.indexOf('.css') !== -1
```

Comment:
Checking the file extension by using `String.indexOf` will not work for certain filenames, ex: `somefile.jsstyle.css` or `somefile.cssstuff.js`.